static/edx-pl/sass/main-ltr.scss

```diff
@@ -0,0 +1,22 @@
+// ------------------------------
```

Comment:
Also, @feanil, the collectstatic question above is topic I pinged you about today in the devops HipChat room, FYI. Just want to make sure I'm defining the new style bundle in the right place.